docs/book/src/cronjob-tutorial/epilogue.md

```diff
@@ -4,16 +4,17 @@ By this point, we've got a pretty full-featured implementation of the
 CronJob controller, and have made use of most of the features of
 KubeBuilder.
 
+If you want to learn how to write integration tests for the CronJob kind
```

Comment:
The commits need to be squashed. We can have just one for be able to merge.

Reply:
Got it! Squashed commits :)